src/app/components/Select/index.tsx

```diff
@@ -96,6 +97,54 @@ const TertiaryButton = forwardRef(
   },
 )
 
+export const StyledLightSelectButton = styled(StyledSelectButton)(() => ({
+  color: COLORS.grayDark,
+  border: '1px solid',
+  borderColor: COLORS.grayMediumLight,
+  backgroundColor: `${COLORS.white} !important`,
+  [`&.${selectClasses.focusVisible}`]: {
+    backgroundColor: COLORS.white,
+  },
+}))
+
+export const StyledLightSelectListbox = styled(StyledSelectListbox)(() => ({
+  background: COLORS.white,
+}))
+
+export const StyledLightSelectOption = styled(StyledSelectOption)(() => ({
+  color: COLORS.grayDark,
+  [`&:hover:not(.${optionClasses.disabled}),
+  &.${optionClasses.highlighted}`]: {
+    backgroundColor: COLORS.grayMediumLight,
+  },
+  [`&.${optionClasses.disabled}`]: {
+    // backgroundColor: COLORS.white,
+  },
+}))
+
+const LightTertiaryButton = forwardRef(
```

Comment:
This duplication seems unnecessary, I would rather see css changes based on props.

Reply:
I did this now, but I am not sure if that made the code simpler or not.